test/controllers/income/bbsi/BbsiCloseAccountControllerSpec.scala

```diff
@@ -141,7 +140,7 @@ class BbsiCloseAccountControllerSpec extends PlaySpec
 
         val sut = createSUT
 
-        val date = TaxYearResolver.startOfCurrentTaxYear.minusDays(1)
+        val date = TaxYear().endPrev
```

Comment:
Is this the same as TaxYearResolver.startOfCurrentTaxYear.minusDays(1)?

Reply:
Or TaxYear().prev.end